mozci/console/commands/push.py

```diff
@@ -436,6 +457,28 @@ def send_emails(self, total, stats, error_line):
                     f"<error>Failed to send the report by email to address n°{idx} ({email}): {e}</error>"
                 )
 
+    def log_details(self, push, state, expected):
+        total = len(self.regressions[push][state])
+        if not total:
+            return
+
+        differing = []
+        for group in self.regressions[push][state].keys():
+            if (
+                set([c for c, _ in push.group_summaries[group].classifications])
+                != expected
+            ):
```

Comment:
This will be False when we have multiple classifications, and so we'd always count them as wrong.
I'm not sure how to best handle the case where we have multiple classifications.
For now, can you ignore those cases where len(set(...))) is not 1 and just print the number of them?
Then file a follow-up issue to discuss about how to handle them.

CC @ahal @jmaher if you have thoughts.
We can have multiple classifications when, for example, we have a group that is failing consistently in a task and intermittently in another task. It should be super rare I guess... Another case where we have multiple classifications is when there was a mistake in the manual classifications, I guess in those cases ignoring is the best option.

Reply:
What do you mean by multiple classifications?
As for now, if all classifications in `push.group_summaries[group].classifications` are the same and all equal "intermittent" or "fixed by commit" it's working.

For example, if I run the script with:
```
mozci push classify-eval --rev 1a7d29ce984cc34ea7c3932f7b11976794177647 --detailed-classifications
```

And add a few lines to also print the Sheriff classifications for each group:
```
docshell/test/mochitest/mochitest.ini: [('fixed by commit', '80065f2457a7024113287320b483adf15bce8752'), ('fixed by commit', '80065f2457a7024113287320b483adf15bce8752'), [...], ('fixed by commit', '80065f2457a7024113287320b483adf15bce8752'), ('fixed by commit', '80065f2457a7024113287320b483adf15bce8752')]
1 out of 1 real groups were also classified as real by Sheriffs.

dom/serviceworkers/test/mochitest.ini:dom/serviceworkers/test/mochitest-common.ini: [('intermittent', None)]
testing/web-platform/tests/webrtc: [('intermittent', None)]
accessible/tests/mochitest/states/a11y.ini: [('intermittent', None)]
layout/reftests/svg/reftest.list: [('intermittent', None)]
layout/reftests/transform/reftest.list: [('intermittent', None)]
testing/web-platform/tests/IndexedDB: [('intermittent', None), ('intermittent', None)]
layout/reftests/svg/text/reftest.list: [('intermittent', None), ('intermittent', None)]
testing/web-platform/tests/css/printing: [('intermittent', None)]
8 out of 8 intermittent groups were also classified as intermittent by Sheriffs.
```

You can see that some groups have multiple classifications associated to them but since they all are equal it's working as expected.

Reply:
Sorry, I meant when we have multiple classifications and not all of them are equal. E.g. both "fixed by commit" and "intermittent". In this case the set would contain two items.

Reply:
Yes you're right but isn't it intended to be `False` in this case? That's what I understood by reading the issue.
> Then it's simply a matter of checking if GroupSummaries in the real list only have fixed by commit classifications and GroupSummaries in the intermittent list only have intermittent classifications.

Reply:
I just pushed new changes, please tell me if I got it right :smile:

Reply:
I would rather see a log of instances where we have multiple || conflicting classifications.  To me looking at those in detail would be interesting and might lead to some additional logic or acceptance of very low edge cases.

Reply:
We could easily output groups that were missed/have conflicting classifications the same way we are outputting the wrongly classified ones. Please tell me if this is something I should add to this PR :smile:

Reply:
Yeah let's do that